File: Programming Basics/06-Nested Loops - Exercise/numberPyramid.js
function numberPyramid(input) {

    let n = Number(input[0]);
    let string = "";
    let current = 1;
    let isItBigger = false;

    for( let rows = 1; rows <= n; rows++) {
        for(let cols = 1; cols <= rows; cols++) {
            if(current > n) {
                isItBigger = true;
                break;
            }
            string += current + " ";
            current++
            }
            console.log(string);
            string = "";
            if(isItBigger) {
                break;
            }
        }
    } 

numberPyramid(["15"])

File: Programming Basics Java/src/NestedLoopsExercise/NumberPyramid.java
package NestedLoopsExercise;

import java.util.Scanner;

public class NumberPyramid {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int number = Integer.parseInt(scanner.nextLine());
        int current = 1;
        boolean isBigger = false;

        for (int rows = 1; rows <= number; rows++) {

            for (int cols = 1; cols <= rows; cols++) {
                if (current > number) {
                    isBigger = true;
                    break;
                }

                System.out.print(current + " ");
                current++;
            }

            System.out.println();

            if (isBigger) {
                break;
            }
        }
    }
}
